webhooks-extension/src/components/WebhookCreate/tests/WebhookCreate.test.js

```diff
@@ -190,6 +199,25 @@ describe('drop downs should be enabled when a namespace is selected', () => {
     await wait(() => expect(document.getElementById("serviceAccounts").getElementsByClassName("bx--list-box__field").item(0).hasAttribute("disabled")).toBe(false))
 
     });
+  it('service accounts dropdown disabled due to fetch error', async () => {
+    jest.spyOn(API, 'getSecrets').mockImplementation(() => Promise.resolve(secretsResponseMock));
+    const { getByText } = renderWithRouter(
+      <WebhookCreate
+        match={{}}
+        namespaces={namespaces}
+        pipelines={pipelines}
+        fetchPipelines={() => {}}
+        fetchServiceAccounts={() => {}}
+        isFetchingPipelines={false}
+        isFetchingServiceAccounts={false}
+        serviceAccounts={serviceAccounts}
+        getServiceAccountsErrorMessage={"Error fetching service accounts."}
```

Comment:
no need for the braces here

`getServiceAccountsErrorMessage="Error fetching service accounts."`


naming here is also a bit odd, the `get` suggests this should be a function. See my other comment about where this is actually used.
Maybe just `serviceAccountsErrorMessage` would be better?